torchgeo/transforms/transforms.py

```diff
@@ -73,3 +73,100 @@ def forward(self, sample: Dict[str, Tensor]) -> Dict[str, Tensor]:
             sample["boxes"] = sample["boxes"].to(boxes_dtype)
 
         return sample
+
+
+class NCrop(Module):
```

Comment:
Let's keep these private for now (prefix with an underscore, don't include in `__init__`) and try to upstream them to Kornia. Do you want to submit the PR to Kornia or do you want me to?

Reply:
How is this different from https://kornia.readthedocs.io/en/latest/contrib.html#kornia.contrib.extract_tensor_patches? We use this in the inria datamodule.

Reply:
Answering my own question: NCrop is N random crops (better for testing), ExtractTensorPatches is a regular grid (better for evaluation).